### ISSUE TITLE
/lib/ejabberd/priv/bin/ scripts should use bash instead of sh on ubuntu

### ISSUE THREAD

**Issue by szymonpk** [processone/ejabberd#611]
- ejabberd 15.04 (compiled from source with --enable-user=ejabberd --enable-tools --enable-pgsql --enable-zlib --enable-nif --enable-iconv --enable-elixir)
- ubuntu 14.04

While I'm trying to execute `ejabberdctl join_cluster 'ejabberd@<hostname>'`
```
# ejabberdctl join_cluster 'ejabberd@<hostname>'
//lib/ejabberd/priv/bin/joincluster: 18: //lib/ejabberd/priv/bin/joincluster: function: not found
Error: ejabberd@<hostname>
```
If I change header of `/lib/ejabberd/priv/bin/joincluster` to `#!/bin/bash` script will continue fine. It's probably due to sh being linked to dash.

**Comment by weiss** [comment]
This has been fixed in commit 3188941af20c020df756e82393649b561d46c984.

**Comment by szymonpk** [comment]
Thx, sorry!
